src/handwritten/_install_daqmx.py

```diff
@@ -0,0 +1,332 @@
+import click
+import contextlib
+import errno
+import json
+import logging
+import os
+import pathlib
+import subprocess
+import sys
+import tempfile
+import traceback
+from typing import List, Tuple, Optional
+import urllib.request
+
+if sys.platform.startswith("win"):
+   import winreg
+
+MODULE_DIR = pathlib.Path(__file__).parent
+METADATA_FILE = MODULE_DIR / "metadata.json"
+
+def _parse_version(version: str) -> Tuple[int, ...]:
+    """
+    Split the version string into a tuple of integers representing major, minor, and update parts.
+
+    Args:
+        version (str): The version string in the format "major.minor.update".
+
+    Returns:
+        tuple: A tuple of integers representing the major, minor, and update parts of the version.
+
+    >>> _parse_version("23.8.0")
+    (23, 8, 0)
+    >>> _parse_version("24.0.0")
+    (24, 0, 0)
+    >>> _parse_version("invalid_version")
+    Traceback (most recent call last):
+    ...
+    click.exceptions.ClickException: Invalid version format found
+    """
+    try:
+        return tuple(int(part) for part in version.split("."))
+    except ValueError  as e:
+        logging.info("Error: %s", e)
+        logging.debug(traceback.format_exc())
```

Comment:
Don't use the root logger. Add this to the top:
```
_logger = logging.getLogger(__name__)
```

Don't invent new ways to log exceptions. Use `_logger.exception("Failed to parse version.")`. It will automatically find the current exception and log its contents + traceback.

If you want the same behavior at info level, use `_logger.info("Failed to parse version.", exc_info=True)`.

Reply:
ok